sdv/metadata/table.py

```diff
@@ -682,9 +701,16 @@ def reverse_transform(self, data):
         if not self.fitted:
             raise MetadataNotFittedError()
 
+        reversable_columns = [
+            column
+            for column in self._hyper_transformer._output_columns
+            if column in data.columns
+        ]
         try:
-            reversed_data = self._hyper_transformer.reverse_transform(data)
-        except rdt.errors.NotFittedError:
+            reversed_data = self._hyper_transformer.reverse_transform_subset(
+                data[reversable_columns]
+            )
+        except (rdt.errors.NotFittedError, rdt.errors.Error):
```

Comment:
what are the scenarios where we need to catch these other errors

Reply:
Fixed by using `NotFittedError`. Before the `hypertransformer` was always being fitted, and some cases we couldn't reverse transform which raised some `Error` from `RDT`.